code-samples/command-handling/file-setup/13/commands/avatar.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+	data: {
+		name: 'avatar',
+		description: 'Get the avatar URL of the selected user, or your own avatar.',
+		options: [
+			{
+				name: 'target',
+				description: 'The user who\'s avatar to show',
+				type: 'USER',
+			},
+		],
+	},
+	async execute(interaction) {
+		const { user } = interaction.options?.get('target');
```

Comment:
```suggestion
		const { user } = interaction.options.get('target');
```
Do these example's assume the user has already validated `interaction#isCommand()` in the handler? If so, `CommandInteraction#options` isn't nullable, it will just be an empty Collection.

If we don't want to make that assumption, ignore these suggestions.